cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -0,0 +1,49 @@
+{# ==========================================================================
+
+   form-field-with-button.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a form field with text when given:
+
+   settings:  An object used to customize the markup.
+
+   settings.btn_text:          The text for the submit button.
+
+   settings.field.attributes:  Additional attributes to add the input.
```

Comment:
Are there other attributes than `required` that we'd want to add? I'm wondering if that should just be a boolean.